src/catalog/settings-advanced-custom-options.md

```diff
@@ -65,126 +75,11 @@ Adding customizable options to a product is an easy way to offer customers a sel
-                  </tbody>
-               </table>
-            </td>
-         </tr>
-         <tr>
-            <td colspan="2"><b>Select</b></td>
-         </tr>
-         <tr>
-            <td>
-               <p> </p>
-            </td>
-            <td>
-               <table>
-                  <col WIDTH="150">
-                  <col WIDTH="auto">
-                  <tbody>
-                     <tr>
-                        <td>Drop-down</td>
-                        <td>A drop-down list of options. Only one item can be selected at a
-								time. </td>
-                     </tr>
-                     <tr>
-                        <td>Radio Buttons</td>
-                        <td>A set of options that allows only one to be selected at a time.</td>
-                     </tr>
-                     <tr>
-                        <td>Checkbox</td>
-                        <td>A checkbox is a variation of a yes/no option. If the product has more than one checkbox, multiple selections can be made at
-								the same time.</td>
-                     </tr>
-                     <tr>
-                        <td>Multiple Select</td>
-                        <td>A drop-down list of options that accepts multiple selections. To select multiple options, hold down the Ctrl
-								(PC) or Command (Mac) key.</td>
-                     </tr>
-                  </tbody>
-               </table>
-            </td>
-         </tr>
-         <tr>
-            <td colspan="2"><b>Date</b></td>
-         </tr>
-         <tr>
-            <td>
-               <p> </p>
-            </td>
-            <td>
-               <table>
-                  <col WIDTH="150">
-                  <col WIDTH="auto">
-                  <tbody>
-                     <tr>
-                        <td>Date</td>
-                        <td>An input field for a date value. The date can be typed directly
-									into the field, selected from a list or calendar. The method of
-									input used and format of the date is determined by the <a href="{% link design/configuration.md %}">Date &amp; Time Custom Options</a>
-								configuration.</td>
-                     </tr>
-                     <tr>
-                        <td>Date &amp; Time</td>
-                        <td>An input field for date and time values.</td>
-                     </tr>
-                     <tr>
-                        <td>Time</td>
-                        <td>An input field for a time value.</td>
-                     </tr>
-                  </tbody>
-               </table>
-            </td>
-         </tr>
-      </tbody>
-   </table>
+## Input Types ##
+
+|Type|Description|
+|--- |--- |
+|**Text**|An input line or text box where the customer can enter the required information. Options:<br />**Field** - A  single line input field for text.<br />**Area** - A multiple-line input field. Text can be formatted with the WYSIWYG editor, or typed as HTML directly into the text box.|
```

Comment:
The items in the Type column should not be bold

Reply:
OK, I'll fix that. thanks!